ietf/doc/views_search.py

```diff
@@ -345,44 +345,35 @@ def ad_dashboard_group(doc):
         if doc.get_state_slug('draft') == 'rfc':
             return 'RFC'
         elif doc.get_state_slug('draft') == 'active' and doc.get_state_slug('draft-iesg'):
-            return '%s Internet-Draft' % doc.get_state('draft-iesg').name
+            return str(doc.get_state('draft-iesg').name)
```

Comment:
Why are you constructing a str from a str here (and throughout the PR)?

Reply:
Because one of these returns didn't return a string